app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_save :update_posts_counter
+
+  private
+
+  def update_posts_counter
+    posts.increment!(:comments_counter)
+  end
```

Comment:
[OPTIONAL] Please rename the method name with `update_comment_counter` because this is for comment count not for post count so it will look more understandable and please remove `s` from `posts.increment` :+1: check below :point_down: my suggestion :slightly_smiling_face:

 ```suggestion
  def update_comment_counter
    post.increment!(:comments_counter)
  end
```